var/spack/repos/builtin/packages/yoda/package.py

```diff
@@ -63,6 +63,7 @@ class Yoda(AutotoolsPackage):
 
     depends_on('python', type=('build', 'run'))
     depends_on('py-future', type=('build', 'run'))
+    depends_on('zlib', type=('build', 'run'))
```

Comment:
I think this should be:
```suggestion
    depends_on('zlib')
```
since there is no executable that might be used by `yoda` to make `zlib` a run dependency.